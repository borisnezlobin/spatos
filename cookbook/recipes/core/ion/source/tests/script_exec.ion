for script in tests/script_exec/a* tests/script_exec/b*
    target/debug/ion $script
end
