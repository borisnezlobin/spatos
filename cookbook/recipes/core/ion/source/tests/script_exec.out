one one
one twoone
o n e t w o
111 110 101 116 119 111
o n e t w o
firstline secondline
3 2 1
a
3 2 1
1
2
3
4
5
6
1
2
3
4
5
6
a
b
c
d
e
a
b
c
d
e
1
2
3
4
5
6
a
b
c
d
e
1 2 3 4 5 6
a b c d e
1 2 3 4 5 6 a b c d e
1
2
3
1
2
3
1
2
3
1
1 2
1 2 3
4 5
1 2 3 4 5
ğŸ˜‰
ğŸ˜‰
ğŸ˜‰
1 2 3 4 5
pass
1A1 1A2 1B1 1B2
0abc2def5g 0abc2def5h 0abc2def5i 0abc2def6g 0abc2def6h 0abc2def6i 0abc2def7g 0abc2def7h 0abc2def7i 0abc3def5g 0abc3def5h 0abc3def5i 0abc3def6g 0abc3def6h 0abc3def6i 0abc3def7g 0abc3def7h 0abc3def7i 0abc4def5g 0abc4def5h 0abc4def5i 0abc4def6g 0abc4def6h 0abc4def6i 0abc4def7g 0abc4def7h 0abc4def7i 1abc2def5g 1abc2def5h 1abc2def5i 1abc2def6g 1abc2def6h 1abc2def6i 1abc2def7g 1abc2def7h 1abc2def7i 1abc3def5g 1abc3def5h 1abc3def5i 1abc3def6g 1abc3def6h 1abc3def6i 1abc3def7g 1abc3def7h 1abc3def7i 1abc4def5g 1abc4def5h 1abc4def5i 1abc4def6g 1abc4def6h 1abc4def6i 1abc4def7g 1abc4def7h 1abc4def7i
-1 0 1
2 1 0
a b c
d c b
A B C
D C B
-1 0 1
2 1 0
a b c
d c b
A B C
D C B
0 2 4
a c e
A C E
0 -2 -4
e c a
E C A
0 2 4
a c e
A C E
0 -2 -4
e c
E C
1
2
3
4
5
1
2
3
4
5
true
false
foo
bar
